SCANNER_VERSION

```diff
@@ -1 +1 @@
-2.24.0-11-g05cf175999
+2.24.0-42-gc9b46a7736
```

Comment:
Version is based on https://github.com/stackrox/scanner/commit/c9b46a773633a7d1256c97218168a8e6d9c55b99